app/models/user.rb

```diff
@@ -430,13 +439,4 @@ def self.count_all_time_contributor
     revisions = Revision.where(status: 1).pluck(:uid)
     contributors = (notes + answers + questions + comments + revisions).compact.uniq.length
   end
-
-  def self.to_csv(options = {})
```

Comment:
Oh, what happened here?

Reply:
This method is in `application_record.rb` as a class method. All records inherit it.